content/en/docs/collector/getting-started.md

```diff
@@ -200,6 +200,35 @@ sudo rpm -ivh otelcol_{{% param collectorVersion %}}_linux_386.rpm
 {{< /tabpane >}}
 <!-- prettier-ignore-end -->
 
+### Manual Installation
+
+Linux [releases][] are available for various architectures. It's possible to download the
+archive containing the binary and install it on your machine manually:
+
+<!-- prettier-ignore-start -->
+{{< tabpane lang=shell persistLang=false >}}
+{{< tab 386 >}}
+curl -O -L https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_386.tar.gz
```

Comment:
I think we should use `-f` to fail clearly on 404s. It also seems like a good idea to add `--proto '=https --tlsv1.2` to force curl to use HTTPS with TLS v1.2 for increased security (these options are used by e.g. [the Rust official install instructions](https://www.rust-lang.org/tools/install) so I assume they are widely supported).

So on this particular line the curl invocation could be rewritten as:

```suggestion
curl --proto '=https' --tlsv1.2 -fOL https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_386.tar.gz
```

Reply:
I guess this also applies to the existing macOS `curl` invocation, not sure if this is out of scope on this PR though

Reply:
I just copied and pasted the macOS curl command :) i'm happy to change them all in a following PR

Reply:
let's keep it as it is for consistency, but we can have a follow up discussion on the same.

Reply:
https://github.com/open-telemetry/opentelemetry.io/issues/2633